db/migrate/20150702212856_add_before_hello_sign_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddBeforeHelloSignToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :before_hello_sign, :boolean
+    User.all.each do |user|
+      user.update(before_hello_sign: true)
```

Comment:
instead, populate signature_request_id with 'before_hello_sign' for all of the existing peeps